src/main/java/net/skinsrestorer/bukkit/SkinsRestorer.java

```diff
@@ -407,6 +407,11 @@ private void checkBungeeMode() {
             if (!bungeeEnabled) {
                 bungeeEnabled = YamlConfiguration.loadConfiguration(new File("spigot.yml")).getBoolean("settings.bungeecord");
             }
+            if (!bungeeEnabled) {
+                if (new File("paper.yml").exists()) {
```

Comment:
```suggestion
            if (!bungeeEnabled && new File("paper.yml").exists()) {
```
There is no reason to make it span multiple lines